core/src/main/java/org/apache/iceberg/hadoop/ConfigProperties.java

```diff
@@ -23,5 +23,6 @@ public class ConfigProperties {
   private ConfigProperties() {}
 
   public static final String ENGINE_HIVE_ENABLED = "iceberg.engine.hive.enabled";
+  public static final String LOCK_HIVE_ENABLED = "iceberg.lock.hive.enabled";
```

Comment:
This should follow the convention from table properties, like `engine.hive.enabled`, adding the `iceberg` prefix to the table property.

Reply:
Done